app/hotels/domain/exceptions/there_are_no_hotels_exception.py

```diff
@@ -0,0 +1,5 @@
+class ThereAreNoHotelsException(Exception):
```

Comment:
Following the same exception naming that you previously used, I would use HotelsByRatingNotFoundException

Stick to the same name nomenclature!